src/v2/Components/Onboarding/Steps/Artists/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { throttle } from "lodash"
 import React from "react"
 import styled from "styled-components"
-
+import { ProgressIndicator } from "../../../ProgressIndicator"
```

Comment:
Can we make this an absolute path? Generally `../` is ok but anything further obscures a bit.

Reply:
(Also applies to new code elsewhere)